moped-editor/src/views/projects/projectView/ProjectView.js

```diff
@@ -157,36 +171,22 @@ const history = createBrowserHistory();
  */
 const ProjectView = () => {
   const { projectId } = useParams();
-  let query = useQueryParams();
-  const classes = useStyles();
-  const previousFilters = useLocation()?.state?.filters;
+  let [searchParams, setSearchParams] = useSearchParams();
+  const location = useLocation();
+  const activeTab = useActiveTabIndex(searchParams.get("tab"));
+  const previousFilters = location.state?.filters;
```

Comment:
it does not seem completely right that we need to all of `useLocation`, `useParams`, and `useSearchParams`—but i didn't want to wade too far into reworking all this 🤷 

in any case, `useSearchParams` seems to be the ticket—it allows us to update search params and listen to changes just like any other state variable.

Reply:
nice! thanks for making some changes here because I backed away from changing this recently too. Would be great to have a tech debt issue for nav! iirc there are some in-app links that don't use react router too if we haven't squashed them all yet